backend/src/v5/processors/teamspaces/projects/models/commons/tickets.js

```diff
@@ -68,16 +68,17 @@ const storeFiles = (teamspace, project, model, ticket, binaryData) => Promise.al
 	)),
 );
 
-Tickets.addTicket = async (teamspace, project, model, ticket, template) => {
+Tickets.addTicket = async (teamspace, project, model, template, ticket, isFederation) => {
```

Comment:
actually instead of sending `isFederation` everywhere, lets just make a helper function to do a quick query to see if a model is a federation

I know we've been doing this with previous things, but it's a pretty easy query and it'd make the code cleaner

Reply:
should I remove isFederation from all events ?